atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -1091,6 +1095,68 @@ public void testDisableReadWriteConflictChecking() {
         t2.commit();
     }
 
+    @Test
+    public void testGetRowsColumnRangeMultipleIteratorsWorkSafely() {
```

Comment:
Is there a test somewhere in this class that checks the output is correctly ordered? If not, can we add one such test? Even if there is a test in `SnapshotTransaction` there should be one here too, to confirm that the ordering stays the same.

Reply:
Yes (though it's a bit non-obvious to find): `AbstractTransactionTest#testColumnRangePagingTransaction_batchingVisitable()`